pkgs/by-name/li/libretls/package.nix

```
@@ -0,0 +1,53 @@
+{
+  lib,
+  stdenv,
+  fetchgit,
+  autoconf,
+  automake,
+  libtool,
+  openssl,
+  gnumake,
+  makeWrapper,
+  pkg-config,
+}:
+
+let
+  version = "3.8.1";
+in
+stdenv.mkDerivation {
+  pname = "libretls";
+  version = version;
+
+  src = fetchgit {
+    url = "https://git.causal.agency/libretls";
+    tag = version;
+    sha256 = "0xn1ds9d56hf3k723m31gmna0zyy5d0hz7zkiz4gp4ffrfzvsnvh";
+  };
+
+  buildInputs = [
+    autoconf
+    automake
+    gnumake
+    libtool
+    openssl
+  ];
+
+  nativeBuildInputs = [
+    makeWrapper
+    libtool
+    openssl
+    pkg-config
+  ];
+
+  # https://git.causal.agency/libretls/about/
+  updateAutotoolsGnuConfigScriptsPhase = ''
+    autoreconf -fi
+  '';
+
+  meta = {
+    description = "libtls for OpenSSL";
```

```suggestion
    description = "Libtls for OpenSSL";
```

`description` has to be capitalized 
 (https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes)